src/CrudPanel.php

```diff
@@ -237,16 +239,91 @@ public function sort($items)
      *          App/Models/Address defined by a company() method on the user model and an address() method on the
      *          company model, the 'App/Models/Address' string will be returned.
      *
-     * @param $relationString String Relation string. A dot notation can be used to chain multiple relations.
+     * @param string $relationString Relation string. A dot notation can be used to chain multiple relations.
+     * @param int $length Optionally specify the number of relations to omit from the start of the relation string. If
+     *        the provided length is negative, then that many relations will be omitted from the end of the relation
+     *        string.
+     * @param Model $model Optionally specify a different model than the one in the crud object.
```

Comment:
What is it otherwise?

Reply:
If no other model is specified, the model defined in the CrudController is used.

Reply:
Ah, "than the one in the crud object".